javascript/javascript1/week1/js-week1-homework/script.js

```diff
@@ -1,81 +1,112 @@
 function getMyAge() {
-    const yearOfBirth = 1983; //your birth year
-    const yearFuture = 2027; //future year to calculate
-    let age = yearFuture - yearOfBirth; //calculate age
-    //answer result
-    let logString = "You will be " + age + " years old in " + yearFuture; //prepare text to answer
-    
-    alert(logString);
-    console.log(logString);
+  const yearOfBirth = 1983; //your birth year
+  const yearFuture = 2027; //future year to calculate
+  let age = yearFuture - yearOfBirth; //calculate age
+  //answer result
+  let logString = "You will be " + age + " years old in " + yearFuture; //prepare text to answer
```

Comment:
Love the organisational comments! Very good idea. They help you from ending up in this situation:

![image](https://user-images.githubusercontent.com/98409360/205457254-5957a034-0e7a-44f3-867a-a33e2549e394.png)